applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -115,8 +115,11 @@ def GetDefaultSettings(cls):
             "compute_reactions": true,
             "block_builder" : true,
             "builder_and_solver_settings" : {
-                "diagonal_values_for_dirichlet_dofs" : "use_max_diagonal",
-                "silent_warnings"                    : false
+                "consider_lagrange_multiplier_constraint_resolution" : "none",
+                "diagonal_values_for_dirichlet_dofs"                 : "use_max_diagonal",
+                "constraint_scale_factor"                            : "use_mean_diagonal",
```

Comment:
these settings are not there for the "normal" builder and solver...so we cannot use them by default

Reply:
? this is important...

Reply:
this should not be included here IMO but directly be given to the B&S, where it knows its parameters ad can do the validation

Reply:
OK; I will see how can I do to refactor without breaking anything (the problem are as always the checks)